app/views/layouts/application.html.erb

```diff
@@ -16,6 +16,22 @@
     <%= favicon_link_tag asset_pack_path('media/images/govuk-apple-touch-icon-180x180.png'), rel: 'apple-touch-icon', type: 'image/png', size: '180x180' %>
     <%= stylesheet_pack_tag 'application', media: 'all' %>
     <%= javascript_pack_tag 'application', defer: true unless params[:nojs] === "nojs" %>
+
+    <% if cookies[:cookie_consent_1] == "on" %>
+      <script async src="https://www.googletagmanager.com/gtag/js?id=G-LR5YHTJD65"></script>
```

Comment:
I think this is specific to staging. We need to be able to select different scripts for different environments. Happy for this to come later though

Reply:
Ah, the ticket didn't say that. That would seem a bit strange to me, though - why would you want to track users on staging?

Reply:
![image](https://user-images.githubusercontent.com/40488007/112289747-7ae0ef00-8c86-11eb-8349-36e30d5dc1b6.png)

Reply:
🤷 I think it's mostly because we don't know the prod domain